develop-docs/sdk/telemetry/traces/span-links.mdx

```diff
@@ -0,0 +1,221 @@
+function startSpan(options: StartSpanOptions);
+
+interface StartSpanOptions: {
+  //... other options (name, attributes, etc)
+  links?: Link[];
+}
+```
+
+Furthermore, the SDKs need to expose at least an `addLink` method on their respective Span interface. For completeness with OpenTelemetry, ideally they also expose `addLinks`:
+
+```ts {tabTitle:Span API}
+interface Span {
+  // return value can differ depending on platform
+  addLink(link: Link): this;
+  addLinks(links: Link[]): this;
+}
+```
+
+
+
+
+#### Dealing with Negative Sample Rates
+
+## Envelope Item Payload
+
+Span trees are serialized to transaction event envelopes in all Sentry SDKs. Therefore, the envelope item needs to accommodate span links
+in its payload. If the `links` entry is an empty array, it can be omitted from the envelope.
+
+The serialized `links` objects should always contain:
+
+- `span_id: string` - id of the span to link to
+- `trace_id: string` - trace id of the span to link to
+- `sampled: boolean` - required if sampling decision of the span to link to (corresponds to `traceFlags` in Otel span context converted to `boolean`) is available
+- `attributes:` - required if attributes were added to the link
+
+Optionally, the serialized link object can contain further fields from OTel like `traceState`, `isRemote` or `droppedAttributesCount` which will be just forwarded unless we find a use case for them.
+
+The OTel fields `spanId`, `traceId`, and `traceFlags` should be excluded from the links objects in the envelope to avoid duplicate data (e.g. `trace_id`).
+
+If the root span (previously known as transaction) has span links, the links are stored in the trace context.
+
+```ts {tabTitle:Example Trace Context}
+// event envelope item
+{
+  type: "transaction";
+  transaction: string;
+  contexts: {
+    trace: {
+      span_id: string;
+      parent_span_id: string;
+      trace_id: string;
+      // new field for links:
+      links?: Array<{
+        "span_id": string,
+        "trace_id": string,
+        sampled?: boolean, // traceFlags from Otel converted to boolean
+        attributes?: Record<string, AttributeValue>,
+        // + potentially more fields 1:1 from Otel. e.g. (traceState, droppedAttributesCount, isRemote)
+      }>
+      // ...
+    }
+  }
+  // ...
+}
+```
+
+Links that are stored in child spans are serialized in `spans[i].links`.
+
+```ts {tabTitle:Example Span Link Data}
+// event envelope item
+{
+  type: "transaction";
+  transaction: string;
+  spans: Array<{
+    span_id: string;
+    parent_span_id: string;
+    trace_id: string;
+    // new field for links:
+    links?: Array<{
+      "span_id": string,
+      "trace_id": string,
+      sampled?: boolean,
+      attributes?: Record<string, AttributeValue>,
+    }>
+    // ...
+  }>
+  // ...
+}
+```
+
+## Link Types
+
+Links don't require a special meaning or type. When necessary, use the "sentry.link.type" attribute on the link to define the link type.
```

Comment:
l: wdyt about this? Just a slight rewording to make the intention more clear of the sentry.link.type attribute:

```suggestion
Links don't require a special meaning or type but if necessary (e.g. for identifying special links in the product), set the `sentry.link.type` attribute on the link to define the link type.
```